src/containerapp/setup.py

```diff
@@ -47,7 +47,9 @@
 ]
 
 # TODO: Add any additional SDK dependencies here
-DEPENDENCIES = ['pycomposefile>=0.0.29', 'docker']
+DEPENDENCIES = ['pycomposefile>=0.0.29',
+                'docker',
+                'azure-mgmt-kubernetesconfiguration==2.0.0']
```

Comment:
@jiasli May I ask the azure-xxx SDK related dependencies cannot be defined in `setup.py`, right?